src/control/cmd/dmg/system.go

```diff
@@ -172,6 +172,7 @@ type systemStopCmd struct {
 	cmdutil.JSONOutputCmd
 	rankListCmd
 	Force bool `long:"force" description:"Force stop DAOS system members"`
+	Full  bool `long:"full" description:"Attempt a graceful shutdown of DAOS system. Experimental and not for use in production environments"`
```

Comment:
I would have preferred an undocumented option using an environment variable instead of expanding the UI surface in order to make tests pass. This is the software equivalent of adding a button to wreck the transmission of a car with a label that reads, "Don't push this button or you'll wreck your car's transmission." Someone's going to push the button! Because it's there!

Plus, now this flag is cruft that needs to be removed if/when the previous behavior is restored. The main point of my comments about this on the master PR is that there is no benefit to exposing these implementation details to the admin.

Reply:
+1